native/src/components/HorizontalLine.tsx

```diff
@@ -3,6 +3,6 @@ import styled from 'styled-components/native'
 const HorizontalLine = styled.View`
   border-bottom-width: 1px;
   border-color: ${props => props.theme.legacy.colors.textDisabledColor};
-  margin: 20px 0;
+  margin: 8px 0;
```

Comment:
💭 This will cause side effects on other screens, but since we are going to replace this anyway before it hits production its fine.